README.md

```diff
@@ -297,55 +298,68 @@ and it is far easier to work with them when you have deeper understanding yourse
 When it comes to equity compensation, everything depends on understanding some fundamental
 questions:
 
-- What is stock, and how do I own it?
-  What are shares?
-- Is stock the same as equity?
-- What is my equity worth?
-- If I have or will get stock at some point, can I sell it?
+- What is stock, and how does one own it?
+- How does stock relate to shares and equity?
+- What can you do with stock or stock options?
+- What kinds of companies utilize equity compensation?
 
-This requires discussing starting with the basics of how companies and stock work in the
-United States.
-If you think you know all this, you can [jump ahead](#equity-compensation-basics) to
-compensation.
+That last question is often overlooked or misunderstood, and it’s really where everything
+starts.
+
+🄳 For-profit companies that share ownership (in divestments of stock called "shares") in
+exchange for capital or labor are called
+[**stock corporations**](https://www.thebalancesmb.com/what-is-a-stock-corporation-398479). We’ll
```

Comment:
The definitive link for stock corporation is actually the one we had:
https://en.wikipedia.org/wiki/Joint-stock_company

thebalancesmb.com is Google's first match but I don't think it's good enough to merit being the primary link.

Stock corporation and joint-stock company are just alternate terms. We should define them together, probably below.

Possibly we also need a bit of language explaining that company and corporation are essentially synonymous, if that was confusing.